### PR TITLE
added ui element guide docs page and changed component issue template

### LINKED ISSUE #912: Document component requirements
We want a page describing requirements for components. That's intended for contributors, so could be a bit hard to read sometimes.

- [ ] Page should go to `/contribution`.

## High level strategy
* Components should be as user expects them to be. We achieve that by checking refs (link refs).
* Components should be of high quality. We achieve that by manually testing edge cases + refactoring early.

## Core features
* **Visual feedback** - When user interacts with component they should see result of their actions (could be implemented with `:active` selector or somehow else depending on case).
* **Keyboard navigation** (add description).
* **Stateless support** (incorporate StatefulMixin\README.md).

## Bonus points
- [ ] Improve ISSUE_TEMPLATE/comonent-issue-template.md to reflect new understanding + link docs page there. Also link issue create form from docs page.

### PR DESCRIPTION
**Pull request by sergey-kravcov**
close #912 

## Description
Added a new page - ui element guide in Contribution section of the docs.
English and Russian versions of the page were added.
Component issue template was changed. 

Снимок экрана 2021-07-05 в 17 36 07
Снимок экрана 2021-07-05 в 17 36 20

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

File: packages/docs/src/components/page-configs/contribution/ui-element-guide/page-config.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { ApiDocsBlock } from '../../../../types/configTypes'
```
Comment (mxmvshnvsk) [inline packages/docs/src/components/page-configs/contribution/ui-element-guide/page-config.ts:1]: May be should use short aliases for imports? As I know in project supports alias `@/`, that match to `src` folder.

File: packages/docs/src/locales/ru/ru.json [hunk 2]
```diff
@@ -3164,5 +3164,24 @@
...
     "related": "Связанные материалы",
     "styledTablesLink": "Стилизованные таблицы"
+  },
+  "uiElementGuide": {
+    "title": "Требования к элементам UI",
+    "description": "Эта страница предназначена для контрибьюторов vuestic-ui, на ней описаны требования к элементам UI.",
+    "highLevelStrategy": {
+      "title": "Высокоуровневая стратегия",
+      "description": {
+        "userExpectations": "Компоненты должны быть такими, какими их ожидает видеть пользователь. Мы достигаем этого, изучая следующие ссылки ([Vuetify](https://vuetifyjs.com), [Quasar](https://quasar.dev), [Ant Design](https://ant.design), [iView](http://iview.talkingdata.com), [Element UI](https://element.eleme.io), etc.).",
```
Comment (mxmvshnvsk) [inline packages/docs/src/locales/ru/ru.json:3174]: May be should rename `следующие ссылки` -> `следующие проекты`?

File: packages/docs/src/pages/contribution/ui-element-guide.vue [hunk 3]
```diff
@@ -0,0 +1,20 @@
+<template>
+  <docs-content :config="configComputed" />
+</template>
+
+<script lang="ts">
+import { Vue, Options } from 'vue-class-component'
+import DocsContent from '../../components/DocsContent.vue'
+import uiElementGuideConfig from '../../components/page-configs/contribution/ui-element-guide/page-config'
+
+@Options({
```
Comment (mxmvshnvsk) [inline packages/docs/src/pages/contribution/ui-element-guide.vue:10]: Please, add here components name, it really helps to do refactoring with dev tools, #923

**Review by mxmvshnvsk — APPROVED** [review]

**Comment by mxmvshnvsk** [comment]
Thank you!)

**Comment by asvae** [comment]
Very good!

**Pull request merged**